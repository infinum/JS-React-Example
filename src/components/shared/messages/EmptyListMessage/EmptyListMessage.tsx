import React, { FC } from 'react';
import { Center, Heading, Image } from '@chakra-ui/react';

export const EmptyListMessage: FC = () => (
	<Center flexDirection="column" my={8}>
		<Image src="/images/logo-js.png" alt="presentations" width="64px" mb={6} />
		<Heading as="h5" size="md" variant="tertiary">
			Ducky has nothing todo!
		</Heading>
	</Center>
);
