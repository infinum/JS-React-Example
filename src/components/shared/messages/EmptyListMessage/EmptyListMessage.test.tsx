import { axe } from 'jest-axe';

import { render } from 'test-utils';
import { EmptyListMessage } from './EmptyListMessage';

describe('EmptyListMessage', () => {
	it('should matches snapshot', () => {
		const { asFragment } = render(<EmptyListMessage />);

		expect(asFragment()).toMatchSnapshot();
	});

	it('should be accessible', async () => {
		const { container } = render(<EmptyListMessage />);

		const results = await axe(container);

		expect(results).toHaveNoViolations();
	});
});
