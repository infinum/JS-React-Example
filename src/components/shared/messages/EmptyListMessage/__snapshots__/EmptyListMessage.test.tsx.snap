// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyListMessage should matches snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-top: var(--chakra-space-8);
  margin-bottom: var(--chakra-space-8);
}

.emotion-1 {
  width: 64px;
  margin-bottom: var(--chakra-space-6);
}

.emotion-2 {
  font-family: var(--chakra-fonts-heading);
  font-weight: var(--chakra-fontWeights-bold);
  font-size: var(--chakra-fontSizes-xl);
  line-height: 1.2;
}

<div
    class="emotion-0"
  >
    <img
      class="chakra-image emotion-1"
      role="presentation"
      src="/images/logo-js.png"
    />
    <h5
      class="chakra-heading emotion-2"
    >
      Ducky has nothing todo!
    </h5>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;
